addons/udes_stock/utils.py

```diff
@@ -0,0 +1,117 @@
+from urllib.parse import urljoin
+from odoo import _
+
+BASE_PRODUCT_IMAGE_URL = "/web/image/product.product/%i"
+
+
+def location_to_dict(location, **extras):
```

Comment:
I think this method can be removed and used _get_info instead which is defined in udes_get_info module.

Reply:
You're right, although in UDES11 this function is used by workflows with this specific dictionary structure. I am happy to remove it which means when workflows are ported they will need to be adapted or change it to use __get_info to expand.

Reply:
I think yes, is better to use it in that way because thats why we moved get_info and have extras on that so we can call by different variables from different places.

Reply:
As per discussion with Miquel, this will be removed and when workflows are ported a better way will be implemented to get this info. Most surely using __get_info() and not having a utils function just for this.